examples/ecs/generic_systems.rs

```diff
@@ -1,20 +1,26 @@
 use bevy::prelude::*;
 use bevy::ecs::component::Component;
 
-// When entities spawn in call an event
-
 struct A;
 struct B;
 
-fn spawn_entities<T: Component>(mut commands: Commands, q: Query<Entity, With<T>>) {
-    q.for_each(|entity| { commands.entity(entity).spawn();});
-    info!("Entity {:?} spawned in!")
+pub fn spawn_entities<T: Component>(mut commands: Commands,
+    mouse_button_inputs: Res<Input<MouseButton>>,) {
+    for _ in 0..10 {
+        let id = commands.spawn().insert(T).id();
+        if !mouse_button_inputs.just_pressed(MouseButton::Left) {
```

Comment:
I'm not quite sure what's happening here yet, or why we're negating our inputs :)

Reply:
Yes that inputs should not be negated I will fix that.